core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala

```diff
@@ -538,14 +538,7 @@ class SubmissionsService(
 
     for {
       _ <- requireComputePermission(workspaceName)
-
-      // getWorkflowFailureMode early because it does validation and better to error early
-      workflowFailureMode <- getWorkflowFailureMode(submissionRequest)
```

Comment:
The validation portion was included into `SubmissionRequestValidation`. 
Getting the actual value is now done inline, since by the time it is needed, it has already been validated.